### ISSUE TITLE
why 'cost volume' loss is nan?

### ISSUE THREAD

**Issue by huanmx** [JialianW/TraDeS#37]
Hi, thanks for your work.
When I train TraDeS on custom dataset and set batch_size to 32, the 'cost volume' loss is nan. However, when I set the batch_size to 12, the 'cost volume' loss is not nan.
How to solve the issue?

**Comment by JialianW** [comment]
Thanks for the interest.

The loss should not be subject to the batchsize. Can you try to decrease the learning rate and check the annotations and some hyperparamters.

**Comment by huanmx** [comment]
Thanks for replying. I will check it.
